### PR TITLE
Readme Update: updates the Readme to highlight an example using es6 i…

### PR DESCRIPTION
**Pull request by Kiro705**
…mport statements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,12 +18,74 @@ your video content.
...
+import 'videojs-contrib-ads';
+import 'videojs-ima';
+
+var videoOptions = {
+  controls: true,
+  sources: [{
+      src: 'PATH_TO_YOUR_CONTENT_VIDEO',
+      type: 'YOUR_CONTENT_VIDEO_TYPE',
+  }]
+}
```
Comment (gschoppe) [inline README.md]: missing semicolon

File: README.md [hunk 2]
```diff
@@ -18,12 +18,74 @@ your video content.
...
+
+var imaOptions = {
+  adTagUrl: 'https://pubads.g.doubleclick.net/gampad/ads?sz=640x480&iu=/124319096/external/ad_rule_samples&ciu_szs=300x250&ad_rule=1&impl=s&gdfp_req=1&env=vp&output=vmap&unviewed_position_start=1&cust_params=deployment%3Ddevsite%26sample_ar%3Dpreonly&cmsid=496&vid=short_onecue&correlator='
+};
```
Comment (gschoppe) [inline README.md]: Extraneous

Comment (Kiro705) [reply README.md]: Removed code.

File: README.md (hunk 1 above)
Comment (Kiro705) [reply README.md]: Added semicolon

**Pull request merged**